tests/test_filters.py

```diff
@@ -67,24 +67,42 @@ def test_clip_filter(datasets):
             else:
                 assert isinstance(clp, pyvista.UnstructuredGrid)
 
-@skip_windows
+
+# @skip_windows
 @skip_mac
-def test_clip_by_scalars_filter(datasets):
+@pytest.mark.parametrize('both', [False, True])
+@pytest.mark.parametrize('invert', [False, True])
+def test_clip_by_scalars_filter(both, invert):
     """This tests the clip filter on all datatypes available filters"""
+    datasets = [Sphere()]
```

Comment:
I think you should add back all the datasets to the test.  The clip filters use two different vtk algorithms depending on dataset type, so it is important that more than 1 dataset is tested here.